types/stripe/index.d.ts

```diff
@@ -9176,6 +9104,69 @@ declare namespace Stripe {
         id: string;
     }
 
+    interface ICardHashInfo {
+        /**
+         * The card number
+         */
+        number?: string;
+        /**
+         * Card brand. Can be Visa, American Express, MasterCard, Discover, JCB, Diners Club, or Unknown.
+         */
+        brand: "Visa" | "American Express" | "MasterCard" | "Discover" | "JCB" | "Diners Club" | "Unknown" ;
+        exp_month: number;
+        exp_year: number;
+        /**
```

Comment:
nit: can you follow the style of the file and add newlines before these comments?